index.d.ts

```diff
@@ -0,0 +1,66 @@
+/// <reference path="./typings/index.d.ts" />
```

Comment:
I've been playing around with a dummy typescript project which includes your code as "jsonwebtoken" module. However, I couldn't make it work with this line.

Side comment, shouldn't this include `/// <reference types="node" />` since you are using `Buffer` which is defined in the node types?